README.md

```diff
@@ -73,7 +73,14 @@
             <sub><b>Сергей</b></sub>
         </a>
     </td></tr>
-
+<tr>
```

Comment:
And here we [again](https://github.com/Searge/filmoteka/pull/105#discussion_r767137289) (-: